src/cpp/ext/otel/otel_plugin.h

```diff
@@ -189,10 +189,30 @@ class OpenTelemetryPluginBuilderImpl {
   std::shared_ptr<std::set<absl::string_view>> optional_label_keys_;
 };
 
+class ActivePluginOptionsView;
```

Comment:
Consider moving these types inside of `OpenTelemetryPlugin`.  That would allow us to make more of the types and methods in `OpenTelemetryPlugin` private.

Reply:
Done.